stable/sumologic-fluentd/values.yaml

```diff
@@ -115,6 +115,10 @@ sumologic:
   ## ref: https://github.com/SumoLogic/fluentd-kubernetes-sumologic#options
   excludeUnitRegex: ""
 
+  ## The field name for json formatted sources that should be used as the time
+  ## ref: https://docs.fluentd.org/v0.12/articles/formatter_json#time_key-(string,-optional,-defaults-to-%E2%80%9Ctime%E2%80%9D)
+  # timeKey:
```

Comment:
I feel like we should probably include the default here, instead of leaving it undefined

Reply:
I agree with @frankreno . Defaulting here makes it important to keep the two in sync which can be error prone.

Reply:
I don't disagree, I bring it up because I'd been pushed by chart committers to provide default values. I'm not religious about it, just trying to propagate a style, which the committers seemed to be interested in keeping.